classes/UpgradePage.php

```diff
@@ -269,15 +269,15 @@ private function getJsParams(string $ajaxResult): array
         return [
             'psBaseUri' => __PS_BASE_URI__,
             '_PS_MODE_DEV_' => (defined('_PS_MODE_DEV_') && true == _PS_MODE_DEV_),
-            'PS_AUTOUP_BACKUP' => $this->config->shouldBackupFilesAndDatabase(),
+            UpgradeConfiguration::PS_AUTOUP_BACKUP => $this->config->shouldBackupFilesAndDatabase(),
             'adminDir' => $adminDir,
             'adminUrl' => __PS_BASE_URI__ . $adminDir,
             'token' => $this->token,
             'firstTimeParams' => json_decode($ajaxResult),
             'ajaxUpgradeTabExists' => file_exists($this->autoupgradePath . DIRECTORY_SEPARATOR . 'ajax-upgradetab.php'),
             'currentIndex' => $this->currentIndex,
             'tab' => 'AdminSelfUpgrade',
-            'channel' => $this->config->get('channel'),
+            UpgradeConfiguration::CHANNEL => $this->config->get(UpgradeConfiguration::CHANNEL),
```

Comment:
I guess this file is going to be deleted shortly, but could be worth showing the best way to retrieve the channel

```suggestion
            UpgradeConfiguration::CHANNEL => $this->config->getChannel(),
```